model/single_doc/lexrank_model.py

```diff
@@ -24,7 +24,7 @@ def summarize(self, corpus, queries=None):
         self.assert_summ_input_type(corpus, queries)
 
         documents = [nltk.sent_tokenize(document) for document in corpus]
-        summaries = [self.lxr.get_summary(document, summary_size=self.summary_length, threshold=self.threshold) for
+        summaries = [self.lxr.get_summary(document, summary_size=self.summary_length, threshold=self.threshold)[0] for
```

Comment:
What does this do? Is it getting the first sentence from the summary (which would be wrong), or the first summary from all hypothesis (which is acceptable).